src/main/java/uk/gov/hmcts/reform/prl/models/complextypes/QuarantineLegalDoc.java

```diff
@@ -90,9 +90,10 @@ public class QuarantineLegalDoc {
     public final LocalDateTime scannedDate;
     public final LocalDateTime deliveryDate;
 
+    //PRL-4306- Added confidential category in the exclusion list
     public static String[] quarantineCategoriesToRemove() {
         return new String [] {
-            "citizenQuarantine", "legalProfQuarantine", "cafcassQuarantine", "courtStaffQuarantine"
+            "citizenQuarantine", "legalProfQuarantine", "cafcassQuarantine", "courtStaffQuarantine", "confidential"
```

Comment:
Can you please add test case for the change.

Reply:
Done